scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,123 @@
+        "--repo",
+        type=str,
+        default=default_repo,
+        help=f"Repository name. Defaults to '{default_repo}' (from firebase_github.py)."
+    )
+    parser.add_argument(
+        "--token",
+        type=str,
+        default=os.environ.get("GITHUB_TOKEN"),
+        help="GitHub token. Can also be set via GITHUB_TOKEN environment variable."
+    )
+
+    args = parser.parse_args()
+
+    if not args.token:
+        sys.stderr.write("Error: GitHub token not provided. Set GITHUB_TOKEN environment variable or use --token argument.\n")
+        sys.exit(1)
+
+    # Update the repository details in firebase_github module if different from default
+    if args.owner != firebase_github.OWNER or args.repo != firebase_github.REPO:
+        repo_url = f"https://github.com/{args.owner}/{args.repo}"
+        if not firebase_github.set_repo_url(repo_url):
+            sys.stderr.write(f"Error: Invalid repository URL format for {args.owner}/{args.repo}. Expected format: https://github.com/owner/repo\n")
+            sys.exit(1)
+        # Using print to stderr for info, as absl logging might not be configured here for this script's own messages.
+        print(f"Targeting repository: {firebase_github.OWNER}/{firebase_github.REPO}", file=sys.stderr)
+
+
+    print(f"Fetching review comments for PR #{args.pull_number} from {firebase_github.OWNER}/{firebase_github.REPO}...", file=sys.stderr)
+
+    comments = firebase_github.get_pull_request_review_comments(args.token, args.pull_number)
+
+    if not comments: # This will be true if list is empty (no comments or error in fetching first page)
+        print(f"No review comments found for PR #{args.pull_number}, or an error occurred during fetching.", file=sys.stderr)
+        # If firebase_github.py's get_pull_request_review_comments logs errors, those might provide more details.
+        return # Exit gracefully if no comments
+
+    # Output actual data to stdout
+    print("\n--- Review Comments ---")
+    for comment in comments:
+        user = comment.get("user", {}).get("login", "Unknown user")
+        path = comment.get("path", "N/A")
+        line = comment.get("line", "N/A")
+        body = comment.get("body", "").strip() # Strip whitespace from comment body
+        diff_hunk = comment.get("diff_hunk", "N/A")
+        html_url = comment.get("html_url", "N/A")
+
+        # Only print comments that have a body
+        if not body:
+            continue
+
+        print(f"Comment by: {user}")
+        print(f"File: {path}")
+        # The 'line' field in GitHub's API for PR review comments refers to the line number in the diff.
+        # 'original_line' refers to the line number in the file at the time the comment was made.
+        # 'start_line' and 'original_start_line' for multi-line comments.
+        # For simplicity, we use 'line'.
+        print(f"Line in diff: {line}")
+        print(f"URL: {html_url}")
+        print("--- Diff Hunk ---")
+        print(diff_hunk)
```

Comment:
I think including the entire diff hunk is probably overkill. Maybe add a parameter to specify how much context to include, and include that number of lines leading up to the comment?